src/guardWorker.ts

```
@@ -100,9 +100,37 @@ export const setupGuardWorker = () => {
       return;
     }
 
-    await interaction.deferReply();
-    await executeInteraction(interaction);
+    try {
+      await interaction.deferReply();
+      await executeInteraction(interaction);
+    } catch (error) {
+      // Handle Discord API errors gracefully
+      const discordError = error as any; // Type assertion for the error
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)